src/app/components/login/login.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="container">
+  <img class="image" src="https://blog.vantagecircle.com/content/images/2022/06/motivacion-laboral-.png" alt="MotivationApp">
+  <!-- the image is momentary -->
+  <div class="form">
+    <input class="user" type="text" placeholder="User">
+    <input class="password" type="password" placeholder="Password">
+    <div class="buttons">
+      <button class="login">Login</button>
+      <button class="googleLogin">
+        <img class="googleImage" src="../../../assets/google/googleLogo.png" alt="Google">
+      </button>
+    </div>
+  </div>
+  <span routerLink="register" class="registerTap">Don't You have an acoount?</span>
```

Comment:
The correct element is `<a>` a is for links.
```suggestion
  <a routerLink="register" class="registerTap">Don't You have an acoount?</a>
```